tests/integration/query_e2e_test.py

```
@@ -236,4 +237,24 @@ def test_integration_paid_query_payment_too_high_fails():
         with pytest.raises(PrecheckError, match="failed precheck with status: INSUFFICIENT_PAYER_BALANCE"):
             query.execute(env.client)
     finally:
-        env.close()
\ No newline at end of file
+        env.close()
+
+@pytest.mark.integration
+def test_integration_query_exceeds_max_payment(env):
```

Ruff (0.14.10)

Redefinition of unused `env` from line 12

(F811)